packages/app-admin-ui/client/pages/List/url-state.js

```diff
@@ -97,9 +97,7 @@ const parseFilter = (filter: [string, string], list): FilterType | null => {
   let label;
   const field = list.fields.find(f => {
     if (key.indexOf(f.path) !== 0) return false;
-    const filterType = f.getFilterTypes().find(t => {
-      return key === `${f.path}_${t.type}`;
-    });
+    const filterType = f.parseFilter(key, value);
```

Comment:
@gautamsi  How about this and the encodeFilter(). I think this is  better implement in FieldController for custom filed implementation.  Should i create another pull request solely about this issue?

Reply:
there is no harm in having this in two separate PR.